core/domain/fs_domain.py

```diff
@@ -332,7 +334,8 @@ class DiskBackedFileSystem(object):
 
     Attributes:
         root: str. The path to append to the oppia/ directory.
-        exploration_id: str. The id of the exploration.
+        assets_path: str. The path to the assets folder. It is of the form
```

Comment:
AFAICT, we've never used DiskBackedFileSystem ... let's just delete it?

Reply:
Yeah, ok.

Reply:
Done